modules/fx/arrow-fx-reaktive/src/test/kotlin/arrow/fx/ReaktiveSpec.kt

```diff
@@ -0,0 +1,19 @@
+package arrow.fx
+
+import arrow.test.UnitSpec
+import com.badoo.reaktive.utils.reaktiveUncaughtErrorHandler
+import com.badoo.reaktive.utils.resetReaktiveUncaughtErrorHandler
+import io.kotlintest.Spec
+
+abstract class ReaktiveSpec : UnitSpec() {
+
+  override fun beforeSpec(spec: Spec) {
+    super.beforeSpec(spec)
+    reaktiveUncaughtErrorHandler = {}
```

Comment:
Something is wrong here too still, both in RxJava and Reactor IIRC.

Last I looked into it it looked like a race condition in RxJava where errors were send downstream resulting in `Undeliverable` even when using `emitter.tryError` but haven't dug deeper.

You can reproduce this with the following test if you re-enable the `reaktiveUncaughtErrorHandler`.


```
fun <F> Concurrent<F>.startJoinIsIdentity(EQ: Eq<Kind<F, Int>>, ctx: CoroutineContext): Unit =
    forAll(Gen.int().applicativeError(this)) { fa ->
      fa.fork(ctx).flatMap { it.join() }.equalUnderTheLaw(fa, EQ)
    }
```